packages/go/analysis/ad/esc6.go

```diff
@@ -91,7 +92,7 @@ func PostCanAbuseWeakCertBinding(operation analysis.StatTrackedOperation[analysi
 					} else {
 						for _, dcForNode := range dcForNodes {
 							if strongCertBindingEnforcement, err := dcForNode.Properties.Get(ad.StrongCertificateBindingEnforcementRaw.String()).Int(); err != nil {
-								collector.Collect(fmt.Errorf("error in PostCanAbuseWeakCertBinding: unable to fetch %v property for node ID %v: %v", ad.StrongCertificateBindingEnforcementRaw.String(), dcForNode.ID, err))
+								log.Warnf("error in PostCanAbuseWeakCertBinding: unable to fetch %v property for node ID %v: %v", ad.StrongCertificateBindingEnforcementRaw.String(), dcForNode.ID, err)
```

Comment:
Nice, I was considering making a similar change. This is out of scope for the work in this PR but we probably shouldn't be exiting all of adcs post processing if one of these methods errors

Reply:
I fixed this because it was actively preventing my MR from working with test data